net/dnsproxy/files/dnsproxy.init

```diff
@@ -44,13 +44,20 @@ append_param_bool() {
 load_config_arg() {
 	append_param_bool "$1" "all_servers"
 	append_param_bool "$1" "fastest_addr"
+	append_param_bool "$1" "http3"
 	append_param_bool "$1" "insecure"
 	append_param_bool "$1" "ipv6_disabled"
 	append_param_bool "$1" "refuse_any"
 	append_param_bool "$1" "verbose"
 }
 
 load_config_list() {
+	is_empty "global" "listen_addr" || config_list_foreach "global" "listen_addr" "append_param '--listen'"
+	if $(is_empty "global" "listen_port"); then
+		append_param_arg "global" "listen_port" "--port" "5353"
+	else 
+		config_list_foreach "global" "listen_port" "append_param '--port'"
+	fi
```

Comment:
```suggestion
	if is_empty "global" "listen_addr"; then
		append_param "--listen" "127.0.0.1"
	else 
		config_list_foreach "global" "listen_port" "append_param '--port'"
	fi

	if is_empty "global" "listen_port"; then
		append_param "--port" "5353"
	else 
		config_list_foreach "global" "listen_port" "append_param '--port'"
	fi
```

Addtionally, `config_list_foreach` won't work if the config type is `option`, so we need add a script to migrate old uci configuration.

We can do a simple string replacement:
```sh
#!/bin/sh

[ -s "/etc/config/dnsproxy" ] || exit 0

sed -i -e "s,option listen_addr,list listen_addr,g" -e "s,option listen_port,list listen_port,g" "/etc/config/dnsproxy"
exit 0
```

then put it into ` /etc/uci-defaults/80-dnsproxy-migration`